### DISCUSSION TITLE
Japanese TTS

### DISCUSSION THREAD

**Discussion by lulussi** [killkimno/MORT#6] [Ideas]
Thank you for your hard work, the features provided in MORT are very useful for playing visual novels. I wanted to suggest Japanese TTS for the text before translation if that was possible as it would be helpful as a language learning tool then. Thank you for reading.

**Comment by kmonkeyhead** [comment]
Although it currently supports TTS function.
However, the performance is not good because it is Windows built-in TTS.

You can find it in Translation tab -> TTS.

I keep looking for better TTS
